src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/ProcedureRequestMapper.java

```diff
@@ -66,7 +65,8 @@ private ProcedureRequest mapPatientClinicalInfo(PatientClinicalInfo patientClini
             this.procedureRequest = new ProcedureRequest();
             mapFreeText();
             mapStatus();
-            procedureRequest.setIntent(ProcedureRequestIntent.NULL);
+            // TODO: this is just a valid value used until we know the real one, or how to determine the real one
```

Comment:
Should we attach the ticket number here?

Reply:
Yep!